packages/cli-repl/src/mongocryptd-manager.ts

```diff
@@ -116,8 +144,11 @@ export class MongocryptdManager {
     return proc;
   }
 
-  // Try the passed paths to executables for spawning, and parse the process'es
-  // log to understand on what path/port it is listening on.
+  /**
+   * Try to spawn mongocryptd using the paths passed to the constructor,
+   * and parse the process'es log to understand on what path/port it
```

Comment:
https://forum.wordreference.com/threads/process-or-processs.1704502/ or https://www.google.com/search?q=What+is+the+possessive+form+of+process%3F&sa=X&ved=2ahUKEwiDi7_w69ryAhVLZMAKHc-7DZoQzmd6BAgOEAU&biw=1676&bih=1336 :)